include/Fader.h

```diff
@@ -118,18 +129,31 @@ class LMMS_EXPORT Fader : public QWidget, public FloatModelView
 	void paintEvent(QPaintEvent* ev) override;
 
 	void paintLevels(QPaintEvent* ev, QPainter& painter, bool linear = false);
+	void paintFaderTicks(QPainter& painter);
 
-	int knobPosY() const
-	{
-		float fRange = model()->maxValue() - model()->minValue();
-		float realVal = model()->value() - model()->minValue();
+	float determineAdjustmentDelta(const Qt::KeyboardModifiers & modifiers) const;
+	void adjustModelByDBDelta(float value);
 
-		return height() - ((height() - m_knob.height()) * (realVal / fRange));
-	}
+	int calculateKnobPosYFromModel() const;
+	void setVolumeByLocalPixelValue(int y);
+
+	// Computes the scaled ratio between the maximum dB value supported by the model and the minimum
+	// dB value that's supported by the fader from the given actual dB value.
+	// If the provided input value lies inside the aforementioned interval then the result will be
+	// a value between 0 (value == minimum value) and 1 (value == maximum model value).
+	// If you look at the graphical representation of the fader then 0 represents a point at the bottom
+	// of the fader and 1 a point at the top of the fader.
+	// The ratio is scaled by an internal exponent which is an implementation detail that cannot be
+	// changed for now.
```

Comment:
I would use documentation comments for this (I am hoping that one day we will actually generate the written documentation on a webpage and use it).

```suggestion
	//! Computes the scaled ratio between the maximum dB value supported by the model and the minimum
	//! dB value that's supported by the fader from the given actual dB value.
	//! If the provided input value lies inside the aforementioned interval then the result will be
	//! a value between 0 (value == minimum value) and 1 (value == maximum model value).
	//! If you look at the graphical representation of the fader then 0 represents a point at the bottom
	//! of the fader and 1 a point at the top of the fader.
	//! The ratio is scaled by an internal exponent which is an implementation detail that cannot be
	//! changed for now.
```

Reply:
Good idea! Switched to a Doxygen style block comment with commit 8d5c5230a46.